manifest/provider/plan.go

```diff
@@ -272,6 +271,21 @@ func (s *RawProviderServer) PlanResourceChange(ctx context.Context, req *tfproto
 		return resp, nil
 	}
 
+	ns, err := IsResourceNamespaced(gvk, rm)
```

Comment:
couldn't we do this below, where we already have this logic? https://github.com/hashicorp/terraform-provider-kubernetes/blob/f424b51e9c7f6b1757608e0aad43c73c42d2848b/manifest/provider/plan.go#L56-L57